bindings/wasm/bindings.cpp

```diff
@@ -141,6 +142,21 @@ Manifold Extrude(std::vector<std::vector<glm::vec2>>& polygons, float height,
                            twistDegrees, scaleTop);
 }
 
+std::vector<glm::ivec3> TriangulateJS(
+    std::vector<std::vector<glm::vec2>>& polygons) {
+  CrossSection crossSection(polygons);
```

Comment:
What's the crossSection for?

Reply:
Oh, I forgot to remove this line. Thanks for pointing out.